prog-o-meter.py

```diff
@@ -306,6 +344,25 @@ def read_days_file(_filename):
     days_text.close() 
     return days
 
+def update_log_file(_logname, _log_entry):
+    """ Updates the file [username]_log.txt, adding user's latest update.
+
+    The timestamp is added above the update, with a newline in between. A newline is also added after the update.
+    
+    Args: 
+        _logname: Name of the file to be updated. Should have format [username]_log.txt (username in all lowercase).
+        _log_entry: The text which is to be appended to the file.
+    """
+
+    import time       #Should the time module be imported inside this function, or globally???

```

Comment:
I think the PEP8 standard (a standard styleguide for python) states that imports is done at the top of the file, so I think it would be nice to have all the imports gathered there. Just out of curriosity, do you know if the module time that you are using, differs much from the module datetime, which is already being used elsewhere in the program, as is imported in line 16. I am wondering if we can use the same module in both cases, instead of having two modules with relative similar functions? But I haven't read up on the two modules, maybe they are not as similar as I imagine?

Reply:
I am happy to group all imports at the beginning of the file. Using datetime instead of time is fine.

I have replaced the time.asctime() call on line 359 with datetime.datetime.now().